ui/src/shared/components/AutoRefresh.tsx

```diff
@@ -199,7 +199,7 @@ const AutoRefresh = (
       this.executeQueries()
 
       if (autoRefresh) {
-        this.intervalID = setInterval(this.executeQueries, autoRefresh)
+        this.intervalID = window.setInterval(this.executeQueries, autoRefresh)
```

Comment:
otherwise TypeScript believes that `setInterval` is from Node, instead of the browser. the browser is the api where setInterval is actually being called. if left off here, TypeScript shows a typeerror saying that the type needs to be `NodeJS.Timer`, which is not correct. when I tried to use that type, TypeScript became confused about which type it should be using. This is a known issue with `setInterval` and TypeScript, the solution for which is to use `window` so that the correct API's return type is identified: `number`.